### DISCUSSION TITLE
可以开放下群嘛

### DISCUSSION THREAD

**Discussion by Vason-Hsueh** [shanmiteko/LotteryAutoScript#216] [General]
提了很多issue了，一直想进群

**Comment by shanmiteko** [comment]
我一直以为你是群友(ﾟOﾟ)

**Comment by shanmiteko** [comment]
已开 速进

**Reply by Vason-Hsueh** [reply]
错过啦 呜呜呜

**Reply by Vason-Hsueh** [reply]
可以的话麻烦再开一下吧  邮箱没发通知呜呜呜

**Reply by shanmiteko** [reply]
ok

**Reply by RAINighty** [reply]
我也想进:(

**Reply by shanmiteko** [reply]
> 我也想进:(

话说你们怎么看的这个消息的，点了`watch`?

**Reply by RAINighty** [reply]
> > 我也想进:(
> 
> 话说你们怎么看的这个消息的，点了`watch`?

对啊，watch all activities

**Reply by Vason-Hsueh** [reply]
暗号是啥啊

**Reply by RAINighty** [reply]
> > 我也想进:(
> 
> 话说你们怎么看的这个消息的，点了`watch`?

我qq叫落雨清琅，我加群了，能不能让我进:(

**Reply by shanmiteko** [reply]
> 暗号是啥啊

脚本的公告里有写

**Reply by Vason-Hsueh** [reply]
> > 暗号是啥啊
> 
> 脚本的公告里有写

哪啊 我连群号都是看issue的 呜呜呜

**Reply by RAINighty** [reply]
> > > 暗号是啥啊
> > 
> > 
> > 脚本的公告里有写
> 
> 哪啊 我连群号都是看issue的 呜呜呜

就是脚本运行时会蹦出来的好几排绿色的字

**Reply by RAINighty** [reply]
> > 暗号是啥啊
> 
> 脚本的公告里有写

我加群了，我能不能进：（

**Reply by shanmiteko** [reply]
> > > 暗号是啥啊
> > 
> > 脚本的公告里有写
> 
> 我加群了，我能不能进：（

不是自动审批吗

**Reply by RAINighty** [reply]
> > > > 暗号是啥啊
> > > 
> > > 
> > > 脚本的公告里有写
> > 
> > 
> > 我加群了，我能不能进：（
> 
> 不是自动审批吗

啊？我不会被过滤了吧

**Reply by Vason-Hsueh** [reply]
谢谢 已进群

**Comment by RAINighty** [comment]
![R%{88GS3PN RO3X%EYE5GE](https://user-images.githubusercontent.com/97167694/200007031-06b5a769-06a9-4d1f-8c5e-89f66920669e.png)

**Comment by zbhtzyx** [comment]
找不到群了 呜呜

**Reply by zbhtzyx** [reply]
> 找不到群了 呜呜

大佬可以重新开放一下嘛

**Reply by shanmiteko** [reply]
已开

**Reply by zbhtzyx** [reply]
> 已开

大佬 可以再开一下么。。昨天晚上没搜到。。是不是又关了。。

**Reply by shanmiteko** [reply]
> > 已开
> 
> 大佬 可以再开一下么。。昨天晚上没搜到。。是不是又关了。。

。
